control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -591,3 +591,42 @@ impl From<VolumeSpec> for models::VolumeSpec {
         )
     }
 }
+
+/// In memory representation of volume group.
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct VolumeGroupSpec {
+    /// Name of the volume group.
+    volume_group_name: String,
```

Comment:
```suggestion
    name: String,
```

Reply:
Should we have a type rather than just String?

Reply:
We can but would there be any advantages? It would be always a string I guess?

Reply:
You can use eg type VolumeGroupId = String;
Then if we ever need to add validation for example, we simply replace that with the actual definition we want.

Reply:
Sounds good. Thanks.

Reply:
Do we need a openapi-spec entry for the type so that the rest layer passes of that type or is it fine in the control-plane level?

Reply:
I might be inclined to leave that for another PR and maybe we need to decide if we really need it.